apps/eth/serializers.py

```diff
@@ -136,7 +137,8 @@ class Meta:
     included_serializers = {
         'transaction': TransactionSerializer,
         'transaction_receipt': TransactionReceiptSerializer,
-        'async_job': AsyncJobSerializer
+        'async_job': AsyncJobSerializer,
+        'shipment': FKShipmentSerializer
```

Comment:
Is there a reason why the shipment wasn't also included in the JSONAPIMeta's included resources?

Reply:
Yes, because we don't need to see any of the Shipment's fields on this endpoint, only the relationship.

Reply:
But wouldn't adding it to the Meta's included resources will do the same thing as `include=shipment`, though if you just have `include=shipment` it will only have the shipment included data

Reply:
Right, but since this is a temporary workaround, I don't want to put that in there, because it'll never get taken out.